data/cities.csv

```diff
@@ -19238,3 +19238,166 @@ Zuni,US
 Zurich,US
 Zwingle,US
 Zwolle,US
+Abbotsford,CA
```

Comment:
See previous comment, so many branches :stuck_out_tongue: 

I have a branch with every canadian city added to the csv